test/games/strategy/triplea/delegate/GameDataTestUtil.java

```diff
@@ -80,6 +80,27 @@ public static UnitType carrier(final GameData data) {
     return unitType(Constants.UNIT_TYPE_CARRIER, data);
   }
 
+  // Some units hard coded here rather than placed in Constants at the request of community members Ref: Pull Request 1074
```

Comment:
ie; linux terminals. I do git work on them, and do a number of diffs there - so the  use-case is there. Google did a good bit of research into the guidelines, though found it was a small but incrementally very significant benefit to not require horizontal scrolling.

RE: monitor, it was quite old, but also keep in mind there were bars on the sides for package structure and outline, so it's not as if the full screen real estate was horizontally available. But, case in point, github only shows so many characters on the website.